src/carbEntryScreen.js

```diff
@@ -8,21 +8,94 @@ class CarbEntryScreen {
     async Open() {
         try {
             //assume we are starting from the open screen
-            await match.accessible.Button(CarbsLabel.AddMeal).tap();
+            await match.accessible.Button(text.carbEntryScreen.AddMeal).tap();
```

Comment:
[nit] I see that you've separated the "screen" implementation from the text that is expected to be on the screen. Consider putting anything related to a particular screen (whether it be actions such as `Open` or `Close`, controls such as `CancelHeaderButton`, or text) within the screen class. That way everything for a particular screen is in one place.

In other words, consider moving the contents of `text.carbEntryScreen` into this class, perhaps at the bottom. 

OTOH, I see you are also using "general" text labels, too, like `text.general.Cancel`.

Well, something for you to consider. :)

Reply:
OTTH (On The Third Hand), if your intention of putting all of the text strings into one file (or one location, at least) so that you can easily swap out localizations, then I fully support that. 👍 Might need to change the implementation of `text.***` to something that can dynamically load the correct localization somehow, but that could be done at a later date.

Reply:
yeah it was for localisation purposes.